test/scripts/clusters/test.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ${SCRIPTS_DIR}/lib/debug_functions
+source ${SCRIPTS_DIR}/lib/utils
+
+function verify_gw_topology() {
+    local gateways_count
+    local expected_gateways="${cluster_gateways[$cluster]:-1}"
```

Comment:
I'm not sure if it's a good idea to rely on this being read from file, if the logic for reading is broken this test wont catch it.

It's better to pass this to the function from outside, since we know how many gateways we actually expect.

Reply:
Modified.